skimage/data/__init__.py

```diff
@@ -316,31 +313,6 @@ def lbp_frontal_face_cascade_filename():
     return _fetch('data/lbpcascade_frontalface_opencv.xml')
 
 
-def load(f, as_gray=False):
```

Comment:
Is it easy to add this back? I'm wondering if this might make it hard to backport any pooch fixes. But maybe we release 0.18 quickly?

Reply:
Nevermind, this is the wrapper to the internal `_load` function. should be fine.